driver-core/src/test/resources/initial-dns-seedlist-discovery/txt-record-utf8-character.json

```diff
@@ -1,20 +0,0 @@
-{
```

Comment:
This one is likely to be removed as well.  Can add it back if we work out how to do it.